Gemfile

```diff
@@ -68,6 +68,7 @@ gem 'connect_vbms', git: 'https://github.com/adhocteam/connect_vbms', tag: 'v2.1
 gem 'csv'
 gem 'datadog'
 gem 'date_validator'
+gem 'docx'
```

Comment:
@rmtolmach do you know if there's some sort of vetting process for including gems that haven't previously been included before?

This `docx` gem provides some helper methods on top of the `nogokiri` gem, which is already included.

Reply:
ah, yeah, good call. See the process here: https://depo-platform-documentation.scrollhelp.site/developer-docs/third-party-libraries#Third-partylibraries-Addinglibraries